assets/src/design-system/components/button/index.js

```diff
@@ -114,7 +114,7 @@ const ButtonRectangle = styled(Base)`
   border-radius: ${({ theme }) => theme.borders.radius.small};
 
   padding: ${({ size }) =>
-    size === BUTTON_SIZES.SMALL ? '8px 16px' : '18px 32px'};
+    size === BUTTON_SIZES.SMALL ? '4px 12px' : '18px 32px'};
```

Comment:
Changed this here since apparently there is `border: 2px solid transparent` applied additionally in a separate place which increases the width and height of the button by `4px` making it larger than it's supposed to be. Perhaps this should be addressed in the place where the border is assigned instead:
https://github.com/google/web-stories-wp/blob/446bbd5f30075f8f74884a1f77894749a49b746c/assets/src/design-system/theme/helpers/outline.js#L48

Reply:
Yeah, that sounds like something that should be detracted only when and if the outline is applied.

Reply:
I think I saw somewhere in another PR that this `border: 2px solid transparent;` might/should get removed. Worth synching with pea pod on it.

Reply:
@BrittanyIRL -- do you have any insights regarding this border and its potential removal?

Reply:
Pascal has a great memory, he's right that `border: 2px solid transparent;` in helpers/outline needs to be removed. That got readded in a rebase and snuck through.  here's a small PR to get that style out of here: https://github.com/google/web-stories-wp/pull/6303